src/Tickets/Seating/app/blockEditor/capacity-table.js

```diff
@@ -0,0 +1,42 @@
+import {select} from "@wordpress/data";
+import { storeName } from './store';
+import {addFilter} from "@wordpress/hooks";
+import { getLocalizedString } from '@tec/tickets/seating/utils';
+
+export function filterCapacityTableMappedProps(mappedProps) {
+	const hasSeats = select(storeName).isUsingAssignedSeating();
+	const layoutLocked = select(storeName).isLayoutLocked();
+
+	if ( ! hasSeats || ! layoutLocked ) {
+		return mappedProps;
+	}
+
+	let layoutId  = select(storeName).getCurrentLayoutId();
+	if ( ! layoutId ) {
+		return mappedProps;
+	}
+
+	let seatTypes = select(storeName).getSeatTypesForLayout(layoutId, true);
+	let activeSeatTypes = Object.values( select(storeName).getSeatTypesByPostID() );
+
+	mappedProps.rowsAfter = mappedProps.rowsAfter || [];
+	const seatTypeLabels = activeSeatTypes.map( type => seatTypes[type].name );
+	const seatTypeTotalCapacity = activeSeatTypes.reduce( ( sum, type ) => sum + parseInt(seatTypes[type].seats), 0 );
+	mappedProps.rowsAfter.push({
+		label: getLocalizedString( 'seats-row-label', 'capacity-table' ),
+		items: seatTypeLabels ? `(${seatTypeLabels})` : '',
+		right: String(seatTypeTotalCapacity),
+	});
+
+	mappedProps.totalCapacity  = ( mappedProps.totalCapacity - mappedProps.sharedCapacity ) + seatTypeTotalCapacity;
+	mappedProps.sharedCapacity = '';
+	mappedProps.sharedTicketItems = '';
+
+	return mappedProps;
+}
+
+addFilter(
```

Comment:
Move this to the `filters.js` file.
In that file import the `filterCapacityTableMappedProps` function and hook on the filter.